src/signatures.rs

```diff
@@ -1277,6 +1280,10 @@ signatures! {
     format = Bzip2
     value = b"BZh"
 
+    format = Commodore64Program
+    value = b"\x01\x08"
```

Comment:
Several `value =` means "or".

The syntax for an "and" is done using comma, e.g.: `value = b"\x01\x08", b"\x9E" offset = 6`.